### PR TITLE
Add a 'get-report-metadata' command to allow debugging metadata collection

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

File: data/bash-completion/fwupdtool [hunk 1]
```diff
@@ -22,6 +22,7 @@ _fwupdtool_cmd_list=(
 	'get-history'
 	'get-plugins'
 	'get-remotes'
+	'get-report-metadata'
```
Comment (superm1) [inline data/bash-completion/fwupdtool:25]: should this be get-device-report-metadata maybe since it's focused on one device?

Comment (superm1) [reply data/bash-completion/fwupdtool:25]: Yeah that's exactly my worry.

Comment (hughsie) [reply data/bash-completion/fwupdtool:25]: `get-device-report-metadata` breaks the `get-dev<tab>` autocomplete for me -- what about just `device-metadata` or `device-report-metadata`? Or `report-metadata-device` perhaps?

Comment (superm1) [reply data/bash-completion/fwupdtool:25]: Or how about you just build a report for all metadata instead of just one device?

Comment (hughsie) [reply data/bash-completion/fwupdtool:25]: Actually that's not insane; most of the time I'm using --plugins to filter anyway.

Comment (hughsie) [reply data/bash-completion/fwupdtool:25]: Something like ^^?

**Review by superm1 — DISMISSED** [review]

File: src/fu-tool.c [hunk 2]
```diff
@@ -3712,6 +3789,13 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Attach to firmware mode"),
 			      fu_util_attach);
+	fu_util_cmd_array_add(cmd_array,
+			      "get-report-metadata",
+			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+			      _("DEVICE-ID|GUID"),
```
Comment (superm1) [inline src/fu-tool.c:3795]: this is no long necessary

Comment (hughsie) [reply src/fu-tool.c]: Fixed, thanks.

**Pull request merged**